pkg/kubectl/util/templates/normalizers.go

```diff
@@ -70,7 +70,7 @@ type normalizer struct {
 
 func (s normalizer) markdown() normalizer {
 	bytes := []byte(s.string)
-	formatted := blackfriday.Markdown(bytes, &ASCIIRenderer{Indentation: Indentation}, 0)
+	formatted := blackfriday.Markdown(bytes, &ASCIIRenderer{Indentation: Indentation}, blackfriday.EXTENSION_NO_INTRA_EMPHASIS);
```

Comment:
I think ";" is not needed and markdown.go includes unnecessary spaces. run "go fmt" to both files.

Reply:
All comments have been addressed in the new commit.

Thanks a lot.

Reply:
I have checked it. Looks good.